src/workflow/ArcanistDiffWorkflow.php

```diff
@@ -1739,6 +1757,35 @@ private function getCommitMessageFromUser() {
   }
 
 
+  private function modifiesPath($paths_to_check, $changes) {
+    foreach ($changes as $changed_file => $contents) {
+      if (preg_match($paths_to_check, $changed_file)) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  /**
+   * get revert plan by prompting developer
+   */
+  private function getRevertPlan() {
+    $ff_prompt = 'Is this diff guarded by a feature flag?';
```

Comment:
For some reason there are a bunch of newlines injected before the prompt.  Any way to avoid that @raviagarwal7?

Reply:
not sure about this

Reply:
Ok will just leave it for now, not a huge deal